### PR TITLE
[stable/grafana] Upgrade to Grafana 7.0.1

### PR DESCRIPTION
**Pull request by steven-sheehy**
No

#### What this PR does / why we need it:
Upgrades Grafana to 7.0.1

#### Which issue this PR fixes
N/A

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 5.0.26
-appVersion: 6.7.3
+version: 5.1.0
```
Comment (cpanato) [inline stable/grafana/Chart.yaml:3]: this should be
```suggestion
version: 6.0.0
```

**Review by cpanato — COMMENTED** [review]
There are any breaking changes if someone upgrades from the 6.7.X version to 7.0.X ?

File: stable/grafana/Chart.yaml (hunk 1 above)
Comment (steven-sheehy) [reply stable/grafana/Chart.yaml:3]: There are some minor breaking changes mentioned in the [release notes](https://github.com/grafana/grafana/releases). I don't think many users will be affected by any of those. I didn't have any issues with my upgrade, for example.

My [previous PR](https://github.com/helm/charts/pull/11705) to upgrade from 5.x to 6.0 was just a minor bump. I don't think it's deserving of a major chart version upgrade, but please let me know if you still think it should be and I can update it.

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**